chai-dom.js

```diff
@@ -137,6 +141,33 @@
     }
   })
 
+  chai.Assertion.overwriteChainableMethod('length',
+    function(_super) {
+      return function(length) {
+        var obj = flag(this, 'object');
+        var actualLength = obj.children ? obj.children.length : obj.length;
+        if (actualLength) {
+          var desc = obj.children ? elToString(obj) : nodeListToString(obj);
```

Comment:
I refrained from doing this for this exact reason - didn't want to broaden the scope of this PR too much. Shall I leave it to you, then?